plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -1898,10 +1900,14 @@ protected StartAnswer execute(StartCommand cmd) {
                                 }
                             }
                             tearDownVm(vmMo);
-                        } else if (!hyperHost.createBlankVm(vmNameOnVcenter, vmInternalCSName, vmSpec.getCpus(), vmSpec.getMaxSpeed().intValue(), getReservedCpuMHZ(vmSpec),
-                                vmSpec.getLimitCpuUse(), (int) (vmSpec.getMaxRam() / ResourceType.bytesToMiB), getReservedMemoryMb(vmSpec), guestOsId, rootDiskDataStoreDetails.first(), false,
-                                controllerInfo, systemVm)) {
-                            throw new Exception("Failed to create VM. vmName: " + vmInternalCSName);
+                        } else {
+                            boolean blankVmCreated = hyperHost.createBlankVm(vmNameOnVcenter, vmInternalCSName, vmSpec.getCpus(), vmSpec.getMaxSpeed().intValue(), getReservedCpuMHZ(vmSpec),
+                                    vmSpec.getLimitCpuUse(), (int) (vmSpec.getMaxRam() / ResourceType.bytesToMiB), getReservedMemoryMb(vmSpec), guestOsId, rootDiskDataStoreDetails.first(), false,
+                                    controllerInfo, systemVm);
+                            if (!blankVmCreated) {
+                                throw new Exception("Failed to create VM. vmName: " + vmInternalCSName);
+                            }
+                            deployAsIs = false;
```

Comment:
Can you please include a log line explaining why not deploying-as-is in this case? Otherwise will look confusion as the StartCommand may receive deployAsIs = true and not honouring it

Reply:
@nvazquez I've added a log message. Please see if there needs to be any change in the text.
Also, I've found that systemvms use the same flow with the default systemvm template, i.e., register a blank VM and then start it. Though deployAsIs was already `false` in that case.

Reply:
Yes - as long a new system VM template is not registered it uses the non-deploy-as-is behaviour

Reply:
@shwstppr possible to consider user input through some strictness flag for deployAsIs, and then fallback to not deploying-as-is case if strictness flag is false?

Reply:
@shwstppr, a small concern here. Since we are starting the VM as not deployAsIs for a VM which is actually a deployAsIs, will there be any problem while applying vApp properties as it is. Can you please test that.

Reply:
Thanks for pointing it out @harikrishna-patnala . There seem an issue vApp appliances. Re-working on it.